ios/engine/KMEI/KeymanEngineDemo/MainViewController.swift

```diff
@@ -15,9 +15,6 @@ class MainViewController: UIViewController, UIAlertViewDelegate, TextViewDelegat
   var textView2: TextField!
   var textView3: UITextView!
 
-  private var keyboardDownloadStartedObserver: NotificationObserver?
-  private var keyboardDownloadCompletedObserver: NotificationObserver?
-  private var keyboardDownloadFailedObserver: NotificationObserver?
```

Comment:
`KeymanEngineDemo` doesn't have any keyboard downloading available.  In the past, before the Settings rework, it may have been possible through the 'picker', but picker-based downloads have long been removed.